### ISSUE TITLE
Moisture level always at 0%

### ISSUE THREAD

**Issue by darkyat** [KristopherKubicki/device-plantlink-direct#4]
Unfortunately the moisture level is always at 0% for me using your implementation. 

The Plantlink Sensor is connected properly and the battery status is accurate, but regardless of the soil type or water needs the moisture level never changes.

I am afraid this code is no longer working with the current SmartThings API?

**Comment by purplespider** [comment]
Did you find any solution to this?

**Comment by darkyat** [comment]
> Did you find any solution to this?

Unfortunately not. I have stopped using Plantlink a long time ago. There were simply too many things that did not work for me as intended.

Also, I am not sure if you can still use Plantlink at all as [their website](https://www.scotts.com/en-us/plantlink) reports:
> Service for Plantlink has been discontinued.

**Comment by purplespider** [comment]
Thanks for the quick response. That's a shame.

Yeh, I was hoping that this project would allow me to continue using my PlantLink sensors, as presumably, it doesn't rely on the PlantLink infrastructure at all.

**Comment by darkyat** [comment]
> Yeh, I was hoping that this project would allow me to continue using my PlantLink sensors, as presumably, it doesn't rely on the PlantLink infrastructure at all.

Fair point. I made the wrong assumption it is using API calls while the whole purpose of this handler is to make a direct connection instead. If you still see this issue, I believe the code needs some changes to work with the current Smartthings API.
